open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java

```diff
@@ -120,6 +125,68 @@ public void eventOccurred(InputEvent event)
         }
     };
 
+    /**
+     * The key listener that will copy the coordinates to the clipboard in
+     * preferred format.
+     */
+    private final DiscreteEventAdapter myAltListener = new DiscreteEventAdapter("Cursor Position", "Copy Current Coordinates",
+            "Copies the current coordinates using the preference set in the settings -> coordinates menu.")
+    {
+
+        private String myCoordLabel;
+
+        @Override
+        public void eventOccurred(InputEvent event)
+        {
+            if (myLocation != null)
+            {
+                    DecimalDegrees latitudeDD = Coordinates.create(DecimalDegrees.class, myLocation.getLatD());
+                    DecimalDegrees longitudeDD = Coordinates.create(DecimalDegrees.class, myLocation.getLonD());
+
+                    DegDecimalMin latitudeDDM = Coordinates.create(DegDecimalMin.class, myLocation.getLatD());
+                    DegDecimalMin longitudeDDM = Coordinates.create(DegDecimalMin.class, myLocation.getLonD());
+
+                    DegreesMinutesSeconds latitudeDMS = Coordinates.create(DegreesMinutesSeconds.class, myLocation.getLatD());
+                    DegreesMinutesSeconds longitudeDMS = Coordinates.create(DegreesMinutesSeconds.class, myLocation.getLonD());
+
+                    StringBuilder builder = new StringBuilder("");
+                    Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+                    
+                    switch (myUnitsRegistry.getPreferredUnits(Coordinates.class).getSimpleName())
+                    {
+                        case "DegreesMinutesSeconds":

```

Comment:
This is unrelated to the merge but we should take the time in the future to clean up random strings like this and create and ENUM of similar strings so we can reuse